### DISCUSSION TITLE
Orleans host in C++ application

### DISCUSSION THREAD

**Discussion by sergbrych** [dotnet/orleans#7915] [Q&A]
Hello.
We try launch silo in c++ application. 
We have one c# dll which use orleans and create silo. We have manage interoperability c++ dll which call c# dll. 
And we have an error like that:

```
fail: Orleans.Runtime.Silo[100439]
      Exception during Silo.Start
      System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.AspNetCore.Hosting, Version=5.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.
```

when try to use OrleansDashboard .

Is MS Orleans work correctly from c++ code?

**Comment by ElanHasson** [comment]
Orleans dashboard requires Asp.net to function and serve pages.

You should add a reference to your project containing your silo.
